### DISCUSSION TITLE
Masterclass das PyLadies Fortaleza

### DISCUSSION THREAD

**Discussion by alynnefs** [apyb/comunidade#110] [Geral]
Boa tarde, galeris
As PyLadies Fortaleza foram convidadas para ministrar uma Masterclass de Cultura Digital da ECA-CCBJ, que terá cachê. Optamos por seguir com o CNPJ da APyB. Para isso, precisamos de algumas informações:

1. CNPJ
2. Telefone (de alguém da APyB)
3. Razão social
4. E-mail (eventos@ resolve?)
5. CNAE
6. Certidão negativa de débitos municipal
7. Certidão negativa de débitos federal

Tem algum lugar onde eu possa pegar essas informações?
Nesse caso, teria desconto dos 10%? (só a nível de informação mesmo, o cachê vai ficar pra comunidade local)

Obrigada ❤️

**Comment by anicelysantos** [comment]
Olá, Alynne!

Sugerimos que vocês dêem uma lida nas informações deste link: https://apyb.python.org.br/guias/como-organizar-eventos/

**Reply by alynnefs** [reply]
Obrigada ❤️
Jules tinha mandado mesmo, acho que o "problema" maior são as certidões negativas. Amanhã vejo se eu mesma consigo tirar, qualquer coisa volto aqui

**Comment by tiidadavena** [comment]
> Boa tarde, galeris As PyLadies Fortaleza foram convidadas para ministrar uma Masterclass de Cultura Digital da ECA-CCBJ, que terá cachê. Optamos por seguir com o CNPJ da APyB. Para isso, precisamos de algumas informações:
> 
> 1. CNPJ
> 2. Telefone (de alguém da APyB)
> 3. Razão social
> 4. E-mail (eventos@ resolve?)
> 5. CNAE
> 6. Certidão negativa de débitos municipal
> 7. Certidão negativa de débitos federal
> 
> Tem algum lugar onde eu possa pegar essas informações? Nesse caso, teria desconto dos 10%? (só a nível de informação mesmo, o cachê vai ficar pra comunidade local)
> 
> Obrigada ❤️

E-mail (eventos@ resolve?): melhor usar o `comunicação@python.org.br` porque o de `eventos...` é da PyBR

**Reply by rougeth** [reply]
Email deveria ser das organizadores do evento (pessoa associada que está fazendo a ponte). A APyB não vai saber o que fazer com e-mails vindo da ECA-CCBJ.

**Reply by alynnefs** [reply]
eles pediram os dois, acredito que pra agilizar a conversa

**Comment by tiidadavena** [comment]
> Boa tarde, galeris As PyLadies Fortaleza foram convidadas para ministrar uma Masterclass de Cultura Digital da ECA-CCBJ, que terá cachê. Optamos por seguir com o CNPJ da APyB. Para isso, precisamos de algumas informações:
> 
> 1. CNPJ
> 2. Telefone (de alguém da APyB)
> 3. Razão social
> 4. E-mail (eventos@ resolve?)
> 5. CNAE
> 6. Certidão negativa de débitos municipal
> 7. Certidão negativa de débitos federal
> 
> Tem algum lugar onde eu possa pegar essas informações? Nesse caso, teria desconto dos 10%? (só a nível de informação mesmo, o cachê vai ficar pra comunidade local)
> 
> Obrigada ❤️

As documentações solicitadas serão enviadas para o seu email.

**Reply by alynnefs** [reply]
eu consegui pegar as certidões, perguntei se realmente são esses documentos, mas ainda não me responderam

**Comment by alynnefs** [comment]
Boa tarde (:

Solicitaram também esses documentos:

- Requerimento da Junta Comercial ou Contrato Social e suas alterações/aditivos, se houver cópia do termo de posse do representante legal, ata da reunião/assembleia que o elegeu ou outro documento legal válido por meio do qual seja possível aferir a legitimidade do
representante. (Consegui o Estatuto, mas entendi que também precisa da ata da última assembleia)

Documentos de Identificação do Titular da ME* e dos demais envolvidos
a. Cópia da identidade civil com foto (RG ou CNH);
b. Cópia do CPF
c. Cópia do comprovante de endereço atualizado
d. Carta de Exclusividade; (Caso seja representado por terceiros)
e. Currículo
f. Dados Bancários Vinculado ao CNPJ contratado

*Por ser associação, também entra nessas documentações. Não sei no nome de quem precisa ser, acredito que de qualquer uma das diretoras

Também preciso do PNG da assinatura, para colocar na Carta de Exclusividade. Link que enviaram com a lista da documentação: https://drive.google.com/drive/folders/1wP83IaP4D31fWH62e-fERI3F_AflUJnd

**Reply by alynnefs** [reply]
Sobre o estatuto, em 2021 o Mazulo tinha mandado para um patrocinador uma versão digitalizada da original (assinada, carimbada, etc), de 26 páginas,  chamada "ATA 14_ESTATUTO.pdf". Ele acredita que ainda esteja no drive, caso ajude. Como é um projeto que envolve dinheiro público, a burocracia acaba sendo maior :(

**Comment by cecivieira** [comment]
@alynnefs Vamos centralizar a conversa sobre esse assunto aqui nesta thread. O e-mail será utilizado apenas para te enviar a documentação solicitada.

Sobre a Carta de Exclusividade, peço que a redija e nos envie para revisão e assinatura. Essa carta de exclusividade serve para dispensar a licitação, então é importante que nela informe os motivos desse serviço ser prestado exclusivamente por nós. Sugiro pedir orientação a sua contratante.

Estamos reunindo a documentação e no aguardo da Carta de Exclusividade.

Qual é o prazo para envio da documentação para o ECA-CCBJ?

**Reply by alynnefs** [reply]
Eles mandaram a Carta de Exclusividade, nesse caso é para que a gente não faça a mesma palestra em outro lugar durante algum tempo. Compartilhei com o e-mail da diretoria (https://docs.google.com/document/d/1jDRAL4FlKJRbX26EA9rg2Fze3E-ynwW1muAFWY8J8Gw/edit?usp=sharing).
Sobre o prazo, eu tinha respondido no email da comunicação, porque tinham mandado os documentos iniciais por lá (my bad). Então teoricamente era até ontem, mas se der pra mandar até amanhã eu acredito que dá certo. A aula mesmo vai ser dia 18 de março.

**Comment by cecivieira** [comment]
@alynnefs Massa
Vamos tentar reunir a documentação e te enviar por e-mail até amanhã.

**Reply by alynnefs** [reply]
![Screenshot_2024-03-06-16-21-36-972_com.google.android.gm.jpg](https://github.com/apyb/comunidade/assets/17454743/4df59171-5f93-47f1-9765-b9d4d8fe1cad)

Trago news
Não precisa da assinatura da APyB na Carta de Exclusividade. E pelo que ela falou, os documentos são da @anicelysantos

**Reply by anicelysantos** [reply]
Incluí o documento ali na pasta compartilhada do Masterclass. Os dados de CNPJ tu pode pegar aqui: https://apyb.python.org.br/guias/como-organizar-eventos/

**Comment by gabubellon** [comment]
Pessoas, uma dúvida. Esse assunto deveria estar sendo tratado aqui no comunidades ? Questiono por já ter alguns links e informações que podem não ser de pertinente estarem aberta a toda a internet e indexável.


Talvez devesse ser movida para o repo interno de assuntos a ApyB

**Reply by anicelysantos** [reply]
Oi, Gabu.
Resolvemos discutir aqui a fim de ajudar posteriormente quando houver situações semelhantes.

**Comment by alynnefs** [comment]
@cecivieira deu certo procurar o documento?

**Reply by alynnefs** [reply]
o email de ontem passou batido, chegaram vários ao mesmo tempo. @anicelysantos tem como mandar os documentos pessoais? No fim a mulher falou que faltava os da @tiidadavena, não entendi se podem ser os dela no lugar dos teus

**Reply by anicelysantos** [reply]
Oi, Alynne.
Respondi mais acima que havia subido os documentos na pasta ApyB que você compartilhou e o CNPJ você poderia pegar na página de manuais.

**Reply by alynnefs** [reply]
Eu vi os da APyB e não vi os teu, perdãooo
Muito obrigada, viu? ❤️❤️
E desculpa a leseira, tô resolvendo muita coisa de evento ao mesmo tempo

**Reply by anicelysantos** [reply]
Está faltando mais algum documento? Se sim, sinaliza aqui só o que falta, por favor.

**Reply by alynnefs** [reply]
@anicelysantos, falta o Estatuto e dados bancários. Estatuto eu vou gerar o PDF da página, vocês não acharam aquele documento assinado, né?

**Reply by alynnefs** [reply]
Pediram o estatuto assinado em cartório, que é assinado. É aquele documento que falei que o @mazulo tinha digitalizado, mas ele não tem mais acesso. Então falta ele e os dados bancários. Deram o prazo final pra segunda em horário comercial

edição: também falaram que a ata da assembleia não está assinada então fica faltando:
- dados bancários
- estatuto assinado
- ata assinada

**Reply by anicelysantos** [reply]
- Dados bancários: pode pegar aqui https://apyb.python.org.br/guias/como-organizar-eventos/ 
- Estatuto assinado e ata assinada: incluí na pasta apyb que você compartilhou.

**Reply by alynnefs** [reply]
Obrigada, deusa ❤️
Fecho essa thread e abro outra pra pedir a nota fiscal ou faço aqui mesmo?

**Reply by anicelysantos** [reply]
Acho que vale você deixar aberta só até encerrar as devolutivas com ela em relação a documentação. Quando estiverem tudo ok, pode fechar. Quanto a nota fiscal, você pode seguir os passos de solicitação descritos aqui: https://apyb.python.org.br/guias/como-organizar-eventos/

**Reply by alynnefs** [reply]
Atualização: o evento aconteceu dia 18 e ainda não validaram o contrato, por isso ainda não foi enviado pra assinatura

**Comment by alynnefs** [comment]
Nota solicitada [aqui](https://github.com/apyb/tarefas/issues/579) <3
A moça responsável disse que o contrato deve ser enviado amanhã
